apps/ledger-live-desktop/src/live-common-set-supported-currencies.js

```diff
@@ -5,6 +5,7 @@ import { setPlatformVersion } from "@ledgerhq/live-common/platform/version";
 setPlatformVersion("1.1.0");
 
 setSupportedCurrencies([
+  "avalanche_c_chain",
```

Comment:
Doesn't hurt, but not sure this is in the scope of this PR.
If you enable it for LLD, you should enable it as well for LLM + CLI + tests.
Or keep this for another PR.

Reply:
I enabled for LLM, CLI and test in an other PR but didn't add this one
These are already merged in https://github.com/LedgerHQ/ledger-live/tree/feat/live-3999-coin-integration-avax